client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/index.tsx

```diff
@@ -0,0 +1,299 @@
+	);
+	const latestAtomicTransfer = useSelect( ( select ) =>
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore Until createRegistrySelector is typed correctly
+		select( SITE_STORE ).getSiteLatestAtomicTransfer( siteId || 0 )
+	);
+	const latestAtomicTransferError = useSelect( ( select ) =>
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore Until createRegistrySelector is typed correctly
+		select( SITE_STORE ).getSiteLatestAtomicTransferError( siteId || 0 )
+	);
+	const productsList = useSelect( ( select ) => select( PRODUCTS_LIST_STORE ).getProductsList() );
+	const requiresUpgrade = useSelect( ( select ) => select( SITE_STORE ).requiresUpgrade( siteId ) );
+
+	const wpcomDomain = site?.URL?.replace( /http[s]*:\/\//, '' );
+	const stagingDomain = wpcomDomain?.replace( /\b\.wordpress\.com/, '.wpcomstaging.com' ) || null;
+
+	const productName =
+		site?.plan?.features?.available?.woop && site?.plan?.features?.available?.woop[ 0 ];
+	const upgradingPlan = productName ? productsList?.[ productName ] : null;
+
+	// Filter the Woop transferring blockers.
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore Until createRegistrySelector is typed correctly
+	const transferringBlockers = eligibilityHolds?.filter(
+		( hold: TransferEligibilityError ) => ! TRANSFERRING_NOT_BLOCKERS.includes( hold.code )
+	);
+
+	const isTransferStuck = latestAtomicTransfer?.is_stuck || false;
+	const isBlockByTransferStatus = latestAtomicTransferError || false;
+
+	// Add blocked-transfer-hold when something is wrong in the transfer status.
+	if (
+		! transferringBlockers?.includes( {
+			code: eligibilityHoldsConstants.BLOCKED_ATOMIC_TRANSFER,
+			message: '',
+		} ) &&
+		( isBlockByTransferStatus || isTransferStuck )
+	) {
+		transferringBlockers?.push( {
+			code: eligibilityHoldsConstants.BLOCKED_ATOMIC_TRANSFER,
+			message: '',
+		} );
+	}
+
+	const transferringDataIsAvailable =
+		typeof transferringBlockers !== 'undefined' &&
+		( typeof latestAtomicTransfer !== 'undefined' ||
+			typeof latestAtomicTransferError !== 'undefined' );
+
+	const isDataReady = transferringDataIsAvailable;
+
+	/*
+	 * the site is Ready to Start when:
+	 * - siteId is defined
+	 * - data is ready
+	 * - does not require an upgrade, based on store `woop` feature
+	 */
+	let isReadyToStart = !! ( siteId && transferringDataIsAvailable && ! requiresUpgrade );
+
+	/*
+	 * Check whether the site transferring is blocked.
+	 * True as default, meaning it's True when requesting data.
+	 */
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore Until createRegistrySelector is typed correctly
+	const isTransferringBlocked =
+		latestAtomicTransfer && ( ! transferringDataIsAvailable || transferringBlockers?.length > 0 );
+
+	// when the site is not Atomic, ...
+	if ( isReadyToStart && ! isAtomicSite ) {
+		isReadyToStart =
+			isReadyToStart &&
```

Comment:
Nit: we just checked if `isReadyToStart` was `true` a few lines above, so this line should be unnecessary.

Reply:
Can you show me the line you're talking about? I think I'm missing something 😸

Reply:
The `isReadyToStart = isReadyToStart &&...` code is wrapped in a `if ( isReadyToStart ...` condition. So the former doesn't need to check for `isReadyToStart &&` again. Does that make sense?

Reply:
Ah, I see what you mean. Thanks for clarifying.

This is code that I ported over from another part of the site but I'll make a note to tweak that at some point. It'll definitely make it easier to ready.

Reply:
Fixed in #63458

Reply:
That was fast, thanks 🙌